cfgov/templates/welcome.html

```diff
@@ -3,14 +3,42 @@
 
 {% block furniture %}
     <div class="content-wrapper">
+
         <h1>{% trans 'Welcome' %} <br/>
-        You've logged in to the consumerfinance.gov editorial tools.	
+        {% if 'build' in request.site.hostname %}
```

Comment:
This check won't work if you e.g. access build from its IP address or via a specific server.